std/haxe/format/JsonPrinter.hx

```diff
@@ -202,13 +202,12 @@ class JsonPrinter {
 		#end
 		addChar('"'.code);
 		var i = 0;
+		var length = s.length;
 		#if hl
 		var prev = -1;
 		#end
-		while (true) {
-			var c = StringTools.fastCodeAt(s, i++);
-			if (StringTools.isEof(c))
```

Comment:
This could change the behavior on targets, which allow null byte in their strings.

Reply:
Yes but `fastCodeAt` says "This operation is not guaranteed to work if `s` contains the `\0` character." anyway.